src/s3fs.cpp

```diff
@@ -3523,26 +3533,56 @@ static bool abort_uncomp_mp_list(uncomp_mp_list_t& list)
   }
   memset(buff, 0, sizeof(buff));
 
-  // confirm
-  while(true){
-    printf("Would you remove all objects? [Y/N]\n");
-    if(NULL != fgets(buff, sizeof(buff), stdin)){
-      if(0 == strcasecmp(buff, "Y\n") || 0 == strcasecmp(buff, "YES\n")){
-        break;
-      }else if(0 == strcasecmp(buff, "N\n") || 0 == strcasecmp(buff, "NO\n")){
-        return true;
+  bool   is_all;
+  time_t expire_time = 0;
+  time_t now_time    = time(NULL);
+
+  if(is_cli){
```

Comment:
Maybe we should mark this as deprecated so we can remove it in the future?  Although given the breakage fixed in #944 it is likely not many people use interactive mode today.

Reply:
I see. Certainly the user seems to suffer.
Currently, let's deprecate the function which operates as CLI mode.
Leave the options themselves and let them do the same thing as --incomplete-mpu-list .